.github/workflows/release-publish-suite-commit.yml

```diff
@@ -29,8 +31,9 @@ jobs:
         run: ./test
 
   persist-new-suite-yml:
+    name: Commit Suite Release YML
     runs-on: ubuntu-latest
-    needs: release-tests
+    needs: e2e-tests
```

Comment:
I didn't catch this last time, but if the e2e test fails then it won't commit? how will we get notified of this?

Reply:
Correct. We don't have a good way to notify us (yet) but the plan is to add auto-opening of issues at some point for those.

Reply:
ok - can you please make sure there's an issue filed in the repo that captures that?